cfgov/prepaid_agreements/views.py

```diff
@@ -97,7 +96,7 @@ def get_support_text():
 
 def index(request):
     query = request.GET.copy()
-    params = dict(iterlists(query))
+    params = dict.items(query)
```

Comment:
```suggestion
    params = request.GET.lists()
```

Because the conversion to `dict` is unnecessary here if we're just going to get its `items` anyway.